concordia/routing.py

```diff
@@ -0,0 +1,14 @@
+from channels.auth import AuthMiddlewareStack
+from channels.routing import ProtocolTypeRouter, URLRouter
+from django.conf.urls import url
+
+from . import consumers
+
+application = ProtocolTypeRouter(
+    {
+        # (http->django views is added by default)
+        "websocket": AuthMiddlewareStack(
+            URLRouter([url("^ws/asset/asset_updates/$", consumers.AssetConsumer)])
```

Comment:
Do we need one for reservations as well or do these multiplex?

Reply:
(I'm leaning “should” but wanted to think about that more than I have time for today)

Reply:
I think I should rename the endpoint. The same consumer is used for both updates and reservations currently. We could change that, but I don't see what the benefit of doing so would be. I think it makes more sense to have one stream of messages and then use the message types to distinguish the types of events - updates, reservations, reservation releases (still need to do the reservation releases bit).

Reply:
That seems reasonable to me since we currently don't have any concept for reserving something other than an asset